auth-agents-common/src/main/java/org/apache/atlas/policytransformer/PersonaCachePolicyTransformer.java

```diff
@@ -104,19 +105,14 @@ public List<AtlasEntityHeader> transform(AtlasEntityHeader atlasPolicy) {
                 header.setAttribute(ATTR_POLICY_CONDITIONS, templatePolicy.getPolicyConditions());
 
                 if (POLICY_SERVICE_NAME_ABAC.equals(policyServiceName)) {
-                    if (policyFilterCriteria != null && !policyFilterCriteria.isEmpty()) {
-                        try {
-                            JsonNode filterCriteriaNode = mapper.readTree(policyFilterCriteria);
-                            if (filterCriteriaNode != null && filterCriteriaNode.get("entity") != null) {
-                                JsonNode entityFilterCriteriaNode = filterCriteriaNode.get("entity");
-                                policyFilterCriteria = entityFilterCriteriaNode.toString();
-                            }
-                        } catch (JsonProcessingException e) {
-                            LOG.error("ABAC_AUTH: PersonaCachePolicyTransformer.transform: parsing filterCriteria failed for policy={}, filterCriteria={}", atlasPolicy.getGuid(), policyFilterCriteria);
-                        }
+                    if (JsonToElasticsearchQuery.parseFilterJSON(policyFilterCriteria, "entity") == null) {
```

Comment:
This is already validated at the create/update request, I think no need to add parsing overhead to transformation layer

Reply:
If you want to keep this, please create static constant for "entity"